<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test_class</class>
 <widget class="QWidget" name="Test_class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="cb_0">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_1">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_2">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_3">
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_4">
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_test">
     <property name="text">
      <string>Test</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
